packages/moonstone/VideoPlayer/MediaControls.js

```diff
@@ -815,11 +815,35 @@ const MediaControlsDecorator = hoc((config, Wrapped) => {
 		}
 
 		toggleMoreComponents () {
+			if (!this.state.isFirstReadInfoComponents) {
+				this.focusMoreButton();
+			}
+
 			this.setState((prevState) => {
 				return {
 					showMoreComponents: !prevState.showMoreComponents
 				};
 			});
+
+		}
+
+		focusMoreButton = () => {
+			if (!this.props.visible) {
+				return;
+			}
+
+			// Readout 'more' or 'back' button explicitly.
+			let selectedButton = Spotlight.getCurrent();
+			if (selectedButton === this.mediaControlsNode.querySelector(`.${css.moreButton}`)) {
+				selectedButton.blur();
+				selectedButton.focus();
```

Comment:
Why don't you use Spotlight.blur() and Spotlight.focus() here?

Reply:
This should be Focused for a specific button.
Spotlight.getCurrent(). Focus() is correct, not Spotlight.focus().